src/targets/gpu/multinomial.cpp

```diff
@@ -0,0 +1,25 @@
+#include <migraphx/gpu/multinomial.hpp>
+#include <migraphx/gpu/device/multinomial.hpp>
+#include <migraphx/gpu/context.hpp>
+#include <migraphx/tune_axis.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace gpu {
+
+shape hip_multinomial::compute_shape(std::vector<shape> inputs) const
+{
+    inputs.pop_back();
+    return op.compute_shape(inputs);
```

Comment:
This should require standard shape for inputs.

Reply:
This has been fixed yet.

Reply:
Ah, my bad. Should the check only occur here, or should the `compute_shape` method for the ref version also check for standard shape?

Reply:
The ref version shouldn't need to check, since it should work with non-standard inputs shapes.

Reply:
Okay, got it.